webhook/cert/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
You can't commit a private key, especially on a public repo, it needs to be stored securely.

Reply:
I'll work out a way to test it, although not having a server, that makes it difficult.  I'll see about contacting Foundation, or seeing about Heroku or something similar.  Thanks for the info.  Oh, and I forgot about that server.key...I'll fix. Thanks @sbellone